traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/activities/QuestionsActivity.java

```diff
@@ -161,7 +161,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
             discard.setOnClickListener(b -> {
                 d.dismiss();
                 Intent i = new Intent(this, CatalogItemDetailsActivity.class);
-                i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
                 startActivity(i);
```

Comment:
what happens if user clicks back button while in the new CatalogItemDetailsActivity ?
goes back to Home ? or goes back to the previous CatalogItemDetailsActivity ?

Reply:
Goes back to the previous CatalogItemDetailsActivity. 

I can open an experience (displayed in CatalogItemDetailsActivity). 
Then click on item in 'similar experiences' block. As a result new CatalogItemDetailsActivity with clicked experience will be launched.
If I click 'back' - user will be returned to the previous experience (previous CatalogItemDetailsActivity). I think this is intended behaviour, because navigation order is retained. Is it right?